plugins/inputs/cisco_telemetry_mdt/sample.conf

```diff
@@ -22,8 +22,12 @@
  ## Define (for certain nested telemetry measurements with embedded tags) which fields are tags
  # embedded_tags = ["Cisco-IOS-XR-qos-ma-oper:qos/interface-table/interface/input/service-policy-names/service-policy-instance/statistics/class-stats/class-name"]
 
-  ## Include the delete field in every telemetry message.
-  # include_delete_field = false
+ ## Include the delete field in every telemetry message.
+ # include_delete_field = false
+
+ ## Specify custom name for incoming MDT source field.
+ ## Default is "mdt_source"
+ # source_field_name = "custom_source"
```

Comment:
We usually give the default in the commented field, i.e
```suggestion
 # source_field_name = "mdt_source"
```
Please don't forget to run `make docs` after the change.